### PR TITLE
doc: refinement of timeline use cases

### PR DESCRIPTION
**Pull request by danielsteinkogler**
## Basics

- [ ] I added a line to [changelog.md](/doc/changelog.md)
- [ ] The PR is rebased with current master.
- [ ] Details of what you changed are in commit messages.
- [ ] References to issues, e.g. `close #X`, are in the commit messages and changelog.
- [ ] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added automated tests or a [manual test protocol](../doc/tests/manual/protocol.md)
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)
- [ ] Exceptions to any guidelines are documented

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] I've read through the whole documentation
- [ ] I've checked conformity to guidelines

### CODE REVIEW

File: doc/usecases/assigned/map_timeline_event_view.md [hunk 1]
```diff
@@ -6,23 +6,22 @@
...
-
   - The user sees events represented on the scrollbar of the timeline feature, allowing them to understand how the map changed over time.
     Events include:
-    - addition of an element to the map
-    - removal of an element from the map
-  - The timeline can be scaled, adjusting its start and end date, thus showing more events of the future and the past.
-
+    - addition of an element to the map (colored green)
+    - removal of an element from the map (colored red)
+  - the events a represented by either a line or a bar chart on the sliders.
```
Comment (markus2330) [inline doc/usecases/assigned/map_timeline_event_view.md:20]:
```suggestion
  - the events are represented by a visual indication directly at the day/month/year on the sliders.
```

File: doc/usecases/assigned/map_timeline_event_view.md [hunk 2]
```diff
@@ -6,23 +6,22 @@
...
+  - the timeline can be scaled vertically for better graph readability.
```
Comment (markus2330) [inline doc/usecases/assigned/map_timeline_event_view.md:21]: Rationale usually not needed.

```suggestion
  - the timeline can be scaled vertically for touch screen usage.
```

File: doc/usecases/assigned/map_timeline_single_selection.md [hunk 3]
```diff
@@ -6,18 +6,21 @@
...
 
 ## Scenarios
 
 - **Precondition:**
   - The user has opened the app and selected a map.
-  - The date field shows the current date.
+  - The timeline shows the current date.
 - **Main success scenario:**
-  - The user selects a different date in the date field.
+  - A timeline is presented to the user witch allows them to select year, month and day.
```
Comment (markus2330) [inline doc/usecases/assigned/map_timeline_single_selection.md:17]:
```suggestion
  - A timeline is presented to the user which allows them to select year, month and day.
```

File: doc/usecases/assigned/map_timeline_single_selection.md [hunk 4]
```diff
@@ -6,18 +6,21 @@
...
+  - Year, month and day can be individually selected by scrolling or moving sliders.
```
Comment (markus2330) [inline doc/usecases/assigned/map_timeline_single_selection.md:18]:
```suggestion
  - Year, month and day can be individually selected by scrolling, moving sliders or clicking on the elements.
```

File: doc/usecases/assigned/map_timeline_single_selection.md [hunk 5]
```diff
@@ -6,18 +6,21 @@
...
   - This allows navigation to a different point in the past, present and future.
   - The map updates to show the state of the garden at the selected point in time (removing or adding elements accordingly).
   - Adding or removing elements is done on the selected date.
+  - The selected date is highlighted in bold.
+  - The sliders are synchronized (e.g. if the user scrolls the day slider over the last day of the month, the month slider is updated to the next month)
```
Comment (markus2330) [inline doc/usecases/assigned/map_timeline_single_selection.md:23]: see also unterlagen/english.md in submissions

```suggestion
  - The sliders are synchronized, i.e., if the user scrolls the day slider over the last day of the month, the month slider is updated to the next month
```

File: doc/usecases/assigned/map_timeline_single_selection.md [hunk 6]
```diff
@@ -6,18 +6,21 @@
...
-  - The date field shows the current date.
+  - The timeline shows the current date.
 - **Main success scenario:**
-  - The user selects a different date in the date field.
+  - A timeline is presented to the user witch allows them to select year, month and day.
+  - Year, month and day can be individually selected by scrolling or moving sliders.
   - This allows navigation to a different point in the past, present and future.
   - The map updates to show the state of the garden at the selected point in time (removing or adding elements accordingly).
   - Adding or removing elements is done on the selected date.
+  - The selected date is highlighted in bold.
```
Comment (markus2330) [inline doc/usecases/assigned/map_timeline_single_selection.md:22]:
```suggestion
  - The selected date is highlighted.
```

**Review by markus2330 — CHANGES_REQUESTED** [review]
Great work.
- [x] Please also move the UC to current.
- [x] 0 events should lead to no visual indication, so that 0 and 1 can be easily distinguished.

**Comment by danielsteinkogler** [comment]
@markus2330 i resolved your comments and rebased on current master. i think the PR should be ready to get merged after pipeline succeeds

**Comment by danielsteinkogler** [comment]
and i created some github issues based on the usecases

**Comment by markus2330** [comment]
Great work! :heart:

**Pull request merged**